gateleen-hook/src/main/java/org/swisspush/gateleen/hook/HookHandler.java

```diff
@@ -1305,11 +1308,28 @@ private void registerListener(Buffer buffer) {
         monitoringHandler.updateListenerCount(listenerRepository.size());
     }
 
+    /**
+     * Extract proxyOptions attribute from jsonHook and create a
+     * appropriate property in the hook object.
+     *
+     * This is the same concept as in gateleen-routing:
+     * {@link org.swisspush.gateleen.routing.RuleFactory#setProxyOptions(Rule, JsonObject)}}
+     *
+     * @param jsonHook the json hook
+     * @param hook the hook object
```

Comment:
IMHO those _@param_ descriptions are useless. They only repeat the parameter name. IMHO thats no additional value.

Is there any sense to keep such doc?